lib/SILOptimizer/SILCombiner/SILCombine.cpp

```diff
@@ -732,6 +732,11 @@ static llvm::cl::opt<std::string> CondFailConfigFile(
     "cond-fail-config-file", llvm::cl::init(""),
     llvm::cl::desc("read the cond_fail message strings to elimimate from file"));
 
+static llvm::cl::opt<bool> CondFailConfigFileInterpretAsFunction(
```

Comment:
Is this really needed? There is for sure no overlap in cond_fail message strings and mangled function names. So the file could contain both and the strings are used for both checks.

Reply:
Good point just check the string for a match of either function name or cond_fail message.